hawkular-alerts-rest-tests/pom.xml

```diff
@@ -353,7 +437,14 @@
                 </goals>
                 <configuration>
                   <includes>
-                    <include>**/*ITest*</include>
+                    <include>**/ActionsITest*</include>
+                    <include>**/AlertsITest*</include>
+                    <include>**/ConditionsITest*</include>
+                    <include>**/DampeningITest*</include>
+                    <include>**/EventsITest*</include>
+                    <include>**/EventsLifecycleITest*</include>
+                    <include>**/LifecycleITest*</include>
+                    <include>**/TriggersITest*</include>
```

Comment:
Why did you do this?  It means that if we add a new Itest we'll need to remember to add it here.